README.md

```diff
@@ -7,7 +7,7 @@
 
 # Overview
 
-This solution pack helps community users to submit their custom solution to FSR team.
+The **Submit Your Content** solution pack empowers community members to seamlessly submit their solutions for review on FortiSOAR’s Content Hub. By launching their FortiSOAR instance and leveraging playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions.
```

Comment:
_By launching their FortiSOAR instance and leveraging playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions._

I dont think this is required can be modified to something like users can upload their cusstom solutions like connector widgets and sp's